Dockerfile

```diff
@@ -26,15 +26,20 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
   rjson \
   ggpubr \
   Seurat \
+  colorspace \
   && R -e "source('https://bioconductor.org/biocLite.R')" \
-  && R -e "BiocInstaller::biocLite(c('ensembldb', 'DESeq2', 'qvalue', 'org.Hs.eg.db', 'org.Dr.eg.db', 'ComplexHeatmap', 'ConsensusClusterPlus'), suppressUpdates = TRUE)"
+  && R -e "BiocInstaller::biocLite(c('ensembldb', 'DESeq2', 'qvalue', 'org.Hs.eg.db', 'org.Dr.eg.db', 'ComplexHeatmap', 'ConsensusClusterPlus'), suppressUpdates = TRUE)" \
+  && R -e "BiocInstaller::biocLite(c('scran', 'scater'))"
```

Comment:
Why aren't you suppressing updates in this one? I noticed on your sandbox PR as well.

Reply:
`scater` and `scran` want an updated version of `rlang` and if you suppress updates and don't let them update `rlang` they don't work. There is likely a more sophisticated way to fix this, but this worked for me.

Reply:
I would recommend using [`devtools::install_version`](https://www.rdocumentation.org/packages/devtools/versions/1.13.6/topics/install_version) prior to this step and pinning the version of `rlang` that they want.

Reply:
Pushed this change, but I am building the docker image now to make sure everything still is working fine.

Reply:
Why are you installing from Github rather than CRAN?

Reply:
For rlang? IDK. I could do CRAN. Is that preferred?

Reply:
I don't see why we would want to use the development version of `rlang` and my recommendation above was to use `devtools::install_version` which installs a specified version of a CRAN package.

Reply:
Noted. Will change and push. The docker image appears to have built successfully otherwise.